build_templates/mkmf.template.rttov.intel

```diff
@@ -117,41 +117,37 @@ LD = ifort
 # for more information on installing RTTOV.
 
 # MODIFY THE FOLLOWING VARIABLES FOR YOUR SYSTEM:
-# If your NETCDF, HDFDIR, and RTTOV environment variables are not set,
+# If your NETCDF, HDFEOS5, or RTTOV environment variables are not set,
```

Comment:
This whole section is now very similar to the gfortran version and works on cheyenne with or without the hdf-eos module, as long as the HDFEOS5 variable is set correctly (i.e. consistent with that in the hdf-eos module)